packages/pluggableWidgets/badge-web/src/Badge.editorPreview.tsx

```diff
@@ -4,23 +4,24 @@ import { createElement, ReactElement } from "react";
 import { BadgePreviewProps } from "../typings/BadgeProps";
 import { Badge } from "./components/Badge";
 
-declare function require(name: string): string;
-
 export const preview = (props: BadgePreviewProps): ReactElement => {
-    const { class: classname, style, type, value, brandStyle, onClick } = props;
+    // TODO: Change PIW preview props typing generation to remove the ts-ignore below
+    // @ts-ignore
+    const { className, style, type, value, onClick } = props;
 
     return (
         <Badge
             type={type}
             value={value ? value : ""}
-            clickable={onClick != null}
-            brandStyle={brandStyle}
-            className={classname}
+            onClick={
```

Comment:
what about accessibility?

Reply:
Good point!  I will add `role=button` to the `span` element in `components/Badge` and also make sure the `onClick` will be called when it has focus and the `Enter` or `Space` key is pressed down.

Reply:
Committed work, only thing we need to check are the focus styles. I will work with Chris on that one.